packages/server/src/sdk/app/datasources/plus.ts

```diff
@@ -3,12 +3,33 @@ import {
   DatasourcePlus,
   IntegrationBase,
   Schema,
+  Table,
 } from "@budibase/types"
 import * as datasources from "./datasources"
 import tableSdk from "../tables"
 import { getIntegration } from "../../../integrations"
 import { context } from "@budibase/backend-core"
 
+function checkForSchemaErrors(schema: Record<string, Table>) {
```

Comment:
Do we want to check and make sure first that we haven't got any datasources in prod today that would fail this validation?

Reply:
I've tested this - even if they do exist, they are 100% not supported. Lots of parts of the system simply do not function if a column or table name contains full stops - if they exist they are in a completely broken state already.